docs/syntax.yaml

```diff
@@ -321,8 +320,22 @@ rna:
       reference_nucleotide: the nucleotide at this position in the reference sequence; c
       # ">": the type of change, a substitution
       new_nucleotide: the nucleotide substituting the existing one; g
-
     forms:
       - syntax: sequence_identifier ":" coordinate_type "." position reference_nucleotide ">" new_nucleotide
         examples:
           - NM_004006.3:r.123c>g
+  adjTrx:
```

Comment:
@ahwagner `adjacent_transcript` or similar, please, to keep with the more verbose snake_case used everywhere else.

Reply:
addressed in 759b18b